diff --git a/lib/common/connection.js b/lib/common/connection.js
index 7ea47c65283314724b776818fe1cfb01b2dc8741..785412491d99fa07e74101c034984d2dab6cf985 100644
--- a/lib/common/connection.js
+++ b/lib/common/connection.js
@@ -1145,7 +1145,6 @@ function createMessageConnection(messageReader, messageWriter, _logger, options)
                     responsePromises.delete(id);
                     responsePromise.reject(new messages_1.ResponseError(messages_1.ErrorCodes.MessageWriteError, error.message ? error.message : 'Unknown reason'));
                     logger.error(`Sending request failed.`);
-                    throw error;
                 }
             });
         },
